src/nvim/eval.c

```diff
@@ -6679,7 +6679,9 @@ static struct fst {
   char f_min_argc;              /* minimal number of arguments */
   char f_max_argc;              /* maximal number of arguments */
   void        (*f_func)(typval_T *args, typval_T *rvar);
-  /* implementation of function */
+  /* Implementation of function.
+   * The lines below are matched/used by scripts/gen_events.lua.
+   */
```

Comment:
Added a comment to hopefully avoid issues for this in the future.
But we might also add some assert in genvimvim.lua instead?